### PR TITLE
#1426 Support to configure whether to enabled autoflush

### LINKED ISSUE #1426: Could we disable autoflush permanently

### PR DESCRIPTION
**Pull request by shengjun1985**
Signed-off-by: shengjun.li <shengjun.li@zilliz.com>

**What type of PR is this?**

feature

**What this PR does / why we need it:**

(1) Support to configure whether to enabled auto_flush
(2) change the unit of auto_flush_interval to second

**Which issue(s) this PR fixes:**

Fixes #1426

**Special notes for your reviewer:**

**Does this PR introduce a user-facing change?:**

**Additional documentation (e.g. design docs, usage docs, etc.):**

### CODE REVIEW

File: core/conf/demo/server_config.yaml [hunk 1]
```diff
@@ -46,13 +46,14 @@ server_config:
 #                      | loaded when Milvus server starts up.                       |            |                 |
 #                      | '*' means preload all existing tables.                     |            |                 |
 #----------------------+------------------------------------------------------------+------------+-----------------+
-# auto_flush_interval  | The interval, in milliseconds, at which Milvus             | Integer    | 1000 (ms)       |
-#                      | automatically flushes data to disk.                        |            |                 |
+# auto_flush_interval  | The interval, in seconds, at which Milvus automatically    | Integer    | 1 (s)           |
+#                      | flushes data to disk.                                      |            |                 |
+#                      | Non positive value means disable the regular flush.        |            |                 |
```
Comment (cydrain) [inline core/conf/demo/server_config.yaml]: use "Negative value"

File: core/conf/server_cpu_config.template [hunk 2]
```diff
@@ -46,13 +46,14 @@ server_config:
 #                      | loaded when Milvus server starts up.                       |            |                 |
 #                      | '*' means preload all existing tables.                     |            |                 |
 #----------------------+------------------------------------------------------------+------------+-----------------+
-# auto_flush_interval  | The interval, in milliseconds, at which Milvus             | Integer    | 1000 (ms)       |
-#                      | automatically flushes data to disk.                        |            |                 |
+# auto_flush_interval  | The interval, in seconds, at which Milvus automatically    | Integer    | 1 (s)           |
+#                      | flushes data to disk.                                      |            |                 |
+#                      | Non positive value means disable the regular flush.        |            |                 |
```
Comment (cydrain) [inline core/conf/server_cpu_config.template]: use "Negative value"

File: core/conf/server_gpu_config.template [hunk 3]
```diff
@@ -46,8 +46,9 @@ server_config:
 #                      | loaded when Milvus server starts up.                       |            |                 |
 #                      | '*' means preload all existing tables.                     |            |                 |
 #----------------------+------------------------------------------------------------+------------+-----------------+
-# auto_flush_interval  | The interval, in milliseconds, at which Milvus             | Integer    | 1000 (ms)       |
-#                      | automatically flushes data to disk.                        |            |                 |
+# auto_flush_interval  | The interval, in seconds, at which Milvus automatically    | Integer    | 1 (s)           |
+#                      | flushes data to disk.                                      |            |                 |
+#                      | Non positive value means disable the regular flush.        |            |                 |
```
Comment (cydrain) [inline core/conf/server_gpu_config.template]: use "Negative value"
remember change default value

Comment (cydrain) [inline core/conf/server_gpu_config.template]: Line 56, default value not changed

Comment (shengjun1985) [reply core/conf/server_gpu_config.template]: It is fixed. Thank you.

File: core/conf/demo/server_config.yaml (hunk 1 above)
Comment (shengjun1985) [reply core/conf/demo/server_config.yaml]: Now, we let 0 means disable. Thank you.

**Review by cydrain — APPROVED** [review]

**Pull request merged**